about/index.php

```diff
@@ -3,7 +3,7 @@
 require(BASE . '/api/discord/bots/admin.php');
 //require(BASE . '/api/google/count_majors.php');
 
-head('About Us', true);
+head('About Us', true, false, false, "UNT Robotics is a broad engineering student organization at the University of North Texas. We focus on developing student's skills in engineering &amp; robotics, which involves a range of beginner workshops, industry talks, robotics-based hackathons, recreational projects and competitions.");

```

Comment:
I'm not sure how I feel about having these as function args...

What if we create two new classes:
`HeadingConfig` which contains the auth, return, and the second class `OpenGraphConfig`

That way this ugliness of stacking params should be fixed and instead we'll have a nicer way to specify something like
```
$config = {
  auth: false,
  og: $og_config
}
```

Reply:
Should it go in `template/top.php` or in a separate file in `template/classes/`?

Reply:
Can you instantiate objects in PHP like that? The only way I can see to get a similar instantiation is if it's an array. As an object, it ends up moving the messy stacking of params into a constructor instead of the head function like this:

```php
$og_config = new OpenGraphConfig('UNT Robotics is a broad engineering...', 'images/og-image-2.png');
$config = new HeadingConfig('About Us', true, true, false, $og_config);
head($config);
```

I don't think that's any better than extracting the params into separate variables and passing the variables instead of values.